@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -2197,6 +2216,111 @@ export class TextElementsRenderer {
         return numRenderedTextElements;
     }
 
+    private findLongestStraightPath(
```

Comment:
I know, it's early draft but consider creating this as free-function.
There is no use for `this`
Looks like complicated enough to deserve unit test.